### ISSUE TITLE
Discord-vscode extension failing the extension host with "TypeError: Cannot read property 'largeImageKey' of undefined"

### ISSUE THREAD

**Issue by ComradeNixco** [iCrawl/discord-vscode#99]
Since version 1.28.2 (user setup) of vscode and v2.12.0 of discord-vscode, Ihave this problem when working on an Angular project of mine:
```
C:\Users\Nicolas\.vscode\extensions\ritwickdey.live-sass-3.0.0\node_modules\sasslib\sass.sync.js:61
var Module=typeof Module!=="undefined"?Module:{};var moduleOverrides={};var key;for(key in Module){if(Module.hasOwnProperty(key)){moduleOverrides[key]=Module[key]}}Module["arguments"]=[];Module["thisProgram"]="./this.program";Module["quit"]=(function(status,toThrow){throw toThrow});Module["preRun"]=[];Module["postRun"]=[];var ENVIRONMENT_IS_WEB=false;var ENVIRONMENT_IS_WORKER=false;var ENVIRONMENT_IS_NODE=false;var ENVIRONMENT_IS_SHELL=false;if(Module["ENVIRONMENT"]){if(Module["ENVIRONMENT"]==="WEB"){ENVIRONMENT_IS_WEB=true}else if(Module["ENVIRONMENT"]==="WORKER"){ENVIRONMENT_IS_WORKER=true}else if(Module["ENVIRONMENT"]==="NODE"){ENVIRONMENT_IS_NODE=true}else if(Module["ENVIRONMENT"]==="SHELL"){ENVIRONMENT_IS_SHELL=true}else{throw new Error("Module['ENVIRONMENT'] value is not valid. must be one of: WEB|WORKER|NODE|SHELL.")}}else{ENVIRONMENT_IS_WEB=typeof window==="object";ENVIRONMENT_IS_WORKER=typeof importS

TypeError: Cannot read property 'largeImageKey' of undefined
    at setActivity (C:\Users\Nicolas\.vscode\extensions\icrawl.discord-vscode-2.12.0\src\extension.ts:235:87)
    at Timeout.timers_1.setInterval [as _onTimeout] (C:\Users\Nicolas\.vscode\extensions\icrawl.discord-vscode-2.12.0\src\extension.ts:141:4)
    at ontimeout (timers.js:475:11)
    at tryOnTimeout (timers.js:310:5)
    at Timer.listOnTimeout (timers.js:270:5)
```
I'm not sure if it's a problem of vscode or of the extension or even of mine.
Some light would be appreciated since it completly shut down the Extension host thus not loading or closing most if not all of my other extensions.

**Comment by iCrawl** [comment]
Thanks for reporting this with the error message, that was a regression I introduced in the latest update.

Fixed in `2.12.1` which I just published, should be available soon.

**Comment by ComradeNixco** [comment]
Yes, it solved the problem too, thanks for the fast answer too!
